release/mk-release-snapshots.sh

```diff
@@ -9,23 +9,42 @@
 SQUID_RELEASE=`ls -1 $SQUID_VCS_PATH | cut -d- -f2 | sort -h | tail -n 1`
 
 # Hostname of the CI server where snapshots are built
-SQUID_BUILD_SERVER=buildmaster.squid-cache.org
+SQUID_BUILD_SERVER=build.squid-cache.org
 
 # file Jenkins uses to assemble the list of snapshot files
 outfile=HEAD.out
 
+# arguments:
+# jenkins job name
+# destination directory
+get_artifacts() {
+	local job=$1
+	local outdir=$2
+	local artifactsUrl="https://$SQUID_BUILD_SERVER/job/${job}/lastSuccessfulBuild/artifact/artifacts/*zip*/artifacts.zip"
+	wget --quiet --ca-cert=/etc/ssl/certs/ISRG_Root_X1.pem "$artifactsUrl"
+	if [ ! -e artifacts.zip ] ; then
+		echo "could not download artifacts from $artifactsUrl"
+		return 1
+	fi
+	unzip -qq artifacts.zip
```

Comment:
Please handle unzip errors. Something like this may work:

```suggestion
	unzip -qq artifacts.zip || return $?
```

Reply:
done slightly differently in [108a2a3](https://github.com/squid-cache/ci/pull/26/commits/108a2a34f6c91b7f0a023135032d6a17b506f49b)